Assets/Scripts/Block2Text.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
```

Comment:
Clean up/remove functions such as start, if only to identify a GO, put a comment at the top saying so